source/module.h

```diff
@@ -51,7 +51,18 @@ void PyErr_SetAwsLastError(void);
 /* Set current thread's error indicator based on aws_last_error() and returns NULL */
 PyObject *PyErr_AwsLastError(void);
 
-PyObject *aws_py_memory_view_from_byte_buffer(struct aws_byte_buf *buf, int flags);
+/**
+ * Raise an AWS error corresponding to the current Python error.
+ *
+ * Prints the current Python error to stderr and clears the Python error indicator.
+ * Finds an AWS error code corresponding to the current Python error (fallback is AWS_ERROR_UNKNOWN).
+ * Invokes aws_raise_error() with that error code and always returns AWS_OP_ERR.
+ *
+ * The Python error indicator MUST be set and the GIL MUST be held when calling this function. */
+int aws_raise_py_error(void);
```

Comment:
```aws_py_raise_error```

Reply:
I do not like this naming convention where we start everything with `aws_py_` just because it's in a crt header file. This helper function is only for use in C, and it's a variant on the existing function `aws_raise_error()`, so I named it like the other function.

We were already doing this with functions like `PyErr_SetAwsLastError()` which is a variant on the existing `PyErr_SetString()`

Reply:
renaming this one since they're so similar